### ISSUE TITLE
Make unit of schleem customizable

### ISSUE THREAD

**Issue by Aintaer** [remotelyliving/plumbus-php#6]
Currently Schleem models are always initialized with 10 units of schleem:

https://github.com/remotelyliving/plumbus-php/blob/2dbcbe9e253363f255512d64a86d3bb9fb0e094b/src/Models/Schleem.php#L7

Would like a way of initializing arbitrary amounts of schleem, preferably through an alternate factory method, or a way of providing a customized schleem factory.

**Comment by remotelyliving** [comment]
Fair enough. The plumbus factory should be allowed to order as much or as little schleem as they desire.

**Comment by Aintaer** [comment]
I recommend a schleem pool implementation to promote schleem reuse
